cellprofiler/setting.py

```diff
@@ -1679,7 +1679,7 @@ def test_valid(self, pipeline):
 
 class StructuringElement(Setting):
     def __init__(self, text="Structuring element",
-                 value="disk,1",
+                 value="Disk,1",
```

Comment:
What necessitated this change?

Reply:
This was merely so that the structure elements would show up capitalized in the UI. It's entirely personal preference, so I can revert this if you like.

Reply:
It's okay. In the future, please try to limit changes based on personal preference.